assets/blocks/reader-registration/editor.scss

```diff
@@ -0,0 +1,34 @@
+@import './style.scss';
+
+.newspack-reader-registration {
```

Comment:
It's weird that the block does not get theme's styles, but I'm not sure what we can do about it.

Reply:
I wondered about that too. I intend to continue investigating a better solution for the next step of this block.